server/package.json

```diff
@@ -8,6 +8,9 @@
     "dotenv": "^16.0.1",
     "express": "^4.18.1",
     "http": "^0.0.1-security",
+    "mongodb": "^4.8.0",
+    "mongoose": "^6.4.5",
+    "nodemon": "^2.0.19",
```

Comment:
Please remove the `nodemon` from the dependencies. It will not be used in production. You can either add it in devDependencies or install it in your system. (using `npm i nodemon -g`).